docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -427,3 +427,58 @@ The Concourse CI doesn’t report health metrics through OpenTelemetry. However,
 https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor#span-metrics-processor[OpenTelemetry Collector Span Metrics Processor]
 to derive pipeline execution traces into KPI metrics like throughput and the error rate
 of pipelines.
+
+
+[discrete]
+[[check-service-health-from-cd-pipelines]]
+== Check service health from deployment pipelines
+
+
+Integrating automated service health checks in deployment pipelines is critical for end-to-end deployment automation, which crucially enables deployment frequency to be increased.
+
+Elastic Observability exposes service HTTP health check APIs that can be integrated in deployment pipelines to verify the behaviour of newly deployed instances, and either automatically continue the deployments or rollback according to the health status.
+
+The following example shows a canary deployment pipeline that leverages Elastic health check HTTP APIs to automate the quality check before rolling out the deployment from the canary to the entire set of instances:
+
+image::images/ci-cd-canary-deployment-pipeline.png[Canary Deployment Pipeline]
+
+
+The primary HTTP API to check the health of a service is compare the transaction error rate with a threshold value.
+
+The API call requires authentication, we recomend to use an API Token to authenticate.
+
+The URL is `KIBANA_URL/internal/apm/services`, it is designed to query service metrics.
```

Comment:
```suggestion
The URL is `KIBANA_URL/internal/apm/services`. It is designed to query service metrics.
```

Reply:
These are comma splices. You can't connect two independent clauses with a comma.